text/0000-native-link-modifiers.md

```diff
@@ -83,13 +83,16 @@ but it could be changed later on some future edition boundary.
 
 ### `verbatim`
 
-`+verbatim` means that `rustc` won't add any target-specified library prefixes or suffixes
-(like `lib` or `.a`) to the library name.
+`+verbatim` means that `rustc` itself won't add any target-specified library prefixes or suffixes
+(like `lib` or `.a`) to the library name,
+and will try its best to ask for the same thing from the linker.
 
 For `ld`-like linkers `rustc` will use the `-l:filename` syntax (note the colon)
 when passing the library, so the linker won't add any prefixes or suffixes as well. \
 See [`-l namespec`](https://sourceware.org/binutils/docs/ld/Options.html) in `ld` documentation
-for more details.
+for more details. \
+For linkers not supporting any verbatim modifiers (e.g. `link.exe` or `ld64`)
```

Comment:
`link.exe` is inherently verbatim, and it is *Rust itself* which adds any prefix/suffix. That is a significant difference compared to `ld64`.

Reply:
Not entirely, `link.exe` will add `.obj` to files without extensions.
Also, the paragraph about talks about prefixes/suffixes added by rustc itself.

My idea here was "rustc passes what you passed to the linker without changes and the linker does what it wants with this input", not sure how to word this better.